src/Microsoft.Health.Dicom.Core/Exceptions/MultipleCustomTagsFoundException.cs

```diff
@@ -0,0 +1,18 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.Core.Exceptions
+{
+    /// <summary>
+    /// Exception thrown when two custom tags are found for a given tag path.
+    /// </summary>
+    public class MultipleCustomTagsFoundException : DicomServerException
```

Comment:
When can this happen? Does'nt this mean we allowed bad data?

Reply:
This is part of a bad merge.

---
In reply to: [574701326](https://github.com/microsoft/dicom-server/pull/567#discussion_r574701326) [](ancestors = 574701326)